lib/src/util.dart

```diff
@@ -79,9 +79,10 @@ Stream<T?> streamGenerator<T>(function,
     }
 
     final results = [];
-    await for (final item in function(params: <String, String>{
+    // TODO(SupremeDeity): Confirm if the ?? is necessary
```

Comment:
This should work as intended without further edits but needs confirmation just in case its useless.

Reply:
I can't recall, but is the `before` parameter required if `after` isn't provided? If not, we might be able to just drop it if `beforeFullname` is null:

```dart
<String, String?> {
  'limit': ...,
  if (beforeFullname != null)
    'before': beforeFullName,
}
```